Electron/src/pages/StartMenu/StartMenu.tsx

```diff
@@ -37,6 +44,22 @@ export default function StartMenu({
     password: '',
   });
 
+  const showErrorPopover = ({
+    title,
+    description,
+  }: Pick<PropsInfoPopover, 'title' | 'description'>) => {
```

Comment:
I really like the Pick usage here, good job :). I have to think about using it across the app hehe